src/chartserver/reverse_proxy.go

```diff
@@ -110,8 +112,45 @@ func modifyResponse(res *http.Response) error {
 					hlog.Errorf("failed to handle event: %v", err)
 				}
 			}()
+
+			// Trigger harbor webhook
+			if e != nil && e.Resource != nil && e.Resource.Metadata != nil && len(e.Resource.Metadata.Vtags) > 0 &&
+				len(e.Resource.ExtendedInfo) > 0 {
+				event := &n_event.Event{}
+				metaData := &n_event.ChartUploadMetaData{
+					ChartMetaData: n_event.ChartMetaData{
+						ProjectName: e.Resource.ExtendedInfo["projectName"].(string),
+						ChartName:   e.Resource.ExtendedInfo["chartName"].(string),
+						Versions:    e.Resource.Metadata.Vtags,
+						OccurAt:     time.Now(),
+						Operator:    e.Resource.ExtendedInfo["operator"].(string),
+					},
+				}
+				if err := event.Build(metaData); err != nil {
```

Comment:
This is still incorrect.  Forgot to fix?

Reply:
sorry. ide shows a wrong info when i search the invoking of Build function. i will check carefully again.